docs/iris/src/developers_guide/release.rst

```diff
@@ -1,75 +1,172 @@
 .. _iris_development_releases:
 
 Releases
-********
+========
+
+A release of Iris is a `tag on the SciTools/Iris`_ 
+Github repository.
+
+The summary below is of the main areas that constitute the release.  The final
+section details the :ref:`iris_development_releases_steps` to take.
 
-A release of Iris is a tag on the SciTools/Iris Github repository.
 
 Release branch
-==============
+--------------
 
-Once the features intended for the release are on master, a release branch should be created, in the SciTools/Iris repository.  This will have the name:
+Once the features intended for the release are on master, a release branch 
+should be created, in the SciTools/Iris repository.  This will have the name:
 
     :literal:`{major release number}.{minor release number}.x`
```

Comment:
@tkknight You need to add the leading `v`

Reply:
yes i do